lua/apisix/core/schema.lua

```diff
@@ -45,66 +45,34 @@ local id_schema = {
     }
 }
 
--- todo: chash and roundrobin have different properties, we may support
---     this limitation later.
-
---   {
---     "definitions": {
---       "nodes": {
---         "patternProperties": {
---           ".*": {
---             "minimum": 1,
---             "type": "integer"
---           }
---         },
---         "minProperties": 1,
---         "type": "object"
---       }
---     },
---     "type": "object",
---     "anyOf": [
---       {
---         "properties": {
---           "type": {
---             "type": "string",
---             "enum": [
---               "roundrobin"
---             ]
---           },
---           "nodes": {
---             "$ref": "#/definitions/nodes"
---           }
---         },
---         "required": [
---           "type",
---           "nodes"
---         ],
---         "additionalProperties": false
---       },
---       {
---         "properties": {
---           "type": {
---             "type": "string",
---             "enum": [
---               "chash"
---             ]
---           },
---           "nodes": {
---             "$ref": "#/definitions/nodes"
---           },
---           "key": {
---             "type": "string"
---           }
---         },
---         "required": [
---           "key",
---           "type",
---           "nodes"
---         ],
---         "additionalProperties": false
---       }
---     ]
---   }
+
+local health_checker = {
+    type = "object",
+    properties = {
+        active = {
+            type = "object",
+            properties = {
+                http_path = {type = "string"},
```

Comment:
`http_path` is hard to understand, how about `check_uri`?

Reply:
here is the original define, we need to follow it: https://github.com/Kong/lua-resty-healthcheck/blob/master/lib/resty/healthcheck.lua#L1121